certified-connectors/Projectum PresentIt/apiDefinition.swagger.json

```diff
@@ -0,0 +1,131 @@
+{
+  "basePath": "/",
+  "consumes": [],
+  "definitions": {
+    "GenerationInfo": {
+      "properties": {
+        "dataMap": {
+          "title": "JSON Datamap",
+          "type": "string"
+        },
+        "file": {
+          "format": "binary",
```

Comment:
This is a json object's field and it is not correct to apply 'binary' format here. More likely arbitrary binary content will corrupt json object in a payload.

Please use 'byte' format to work with base64-encoded content or remove the "format" field completely to identify this is a plain string always.

Reply:
As discussed, we will have a rule in swagger validator.